subprojects/platform-play/src/test/groovy/org/gradle/play/plugins/PlayDistributionPluginTest.groovy

```diff
@@ -157,6 +159,13 @@ class PlayDistributionPluginTest extends Specification {
                 }
             })
         }
+
+        where:
+        playVersion | mainClass
+        "2.3.10"    | "play.core.server.NettyServer"
+        "2.4.11"    | "play.core.server.NettyServer"
+        "2.5.18"    | "play.core.server.NettyServer"
```

Comment:
If I'm reading the code in `PlayDistributionPlugin.java` correctly then `NettyServer` is expected for 2.2 and 2.3 and `ProdServerStart` for 2.4+. I'm not sure whether the code or the test is correct, but they don't seem to match?

Reply:
Yes you're right, I guess this is just a typo. I've fixed them, thanks for pointing out!